admin/class-convertkit-pmp-admin.php

```diff
@@ -487,13 +507,25 @@ public function after_all_membership_level_changes( $pmpro_old_user_levels ) {
 			$remove_tags = apply_filters( 'pmpro_convertkit_after_all_membership_level_changes_remove_tags', false, $unsubscribe_tags );
 			
 			if ( ! empty( $remove_tags ) ) {
+				/**
+				 * Allow custom code to filter the unsubscribe tags for the user by email.
+				 *
+				 * @param array $unsubscribe_tags The array of tag IDs to unubscribe this email address from.
```

Comment:
```suggestion
				 * @since TBD
				 *
				 * @param array $unsubscribe_tags The array of tag IDs to unubscribe this email address from.
```

Reply:
We can use whatever version we need here instead of TBD, feel free to update that.